content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/config/org.apache.sling.jcr.repoinit.RepositoryInitializer-default.config

```diff
@@ -0,0 +1,9 @@
+scripts=["
```

Comment:
I'm not 100% sure, but I think I remember repo init only works for >AEM 6.5.

Reply:
Officially we also still support AEM 6.4 SP4+. Was the feature backported for this version?

Reply:
That is why I'm asking :-) I heard "it works with 6.4" but don't know for sure.

Reply:
It works in 6.4 as well, there was a discussion on Twitter about this feature a few days ago.

Reply:
👍